templates/analytics/stats.html

```diff
@@ -1,11 +1,281 @@
+        margin-left: 50px;
+        margin-top: 0px;
+        background: #F0F0F0;
+    }
+    #daily_button {
+        padding: 0px 5px 0px 5px;
+        font-size: 14px;
+        margin-top: 0px;
+        background: #D8D8D8;
+    }
+    #weekly_button {
+        padding: 0px 5px 0px 5px;
+        font-size: 14px;
+        margin-top: 0px;
+        background: #F0F0F0;
+    }
+    #cumulative_button {
+        padding: 0px 5px 0px 5px;
+        font-size: 14px;
+        margin-top: 0px;
+        background: #F0F0F0;
+    }
+    #messages_by_client_realm_button {
+        font-size: 14px;
+        background: #D8D8D8;
+    }
+    #messages_by_client_user_button {
+        font-size: 14px;
+        background: #F0F0F0;
+    }
+    #messages_by_client_cumulative_button {
+        font-size: 14px;
+        margin-left: 50px;
+        background: #D8D8D8;
+    }
+    #messages_by_client_ten_days_button {
+        font-size: 14px;
+        background: #F0F0F0;
+    }
+    #messages_by_client_thirty_days_button {
+        font-size: 14px;
+        background: #F0F0F0;
+    }
+    #messages_by_type_realm_button {
+        font-size: 14px;
+        margin-left: 50px;
+        margin-bottom: 100px;
+        background: #D8D8D8;
+    }
+    #messages_by_type_user_button {
+        font-size: 14px;
+        background: #F0F0F0;
+        margin-bottom: 100px;
+    }
+    #messages_by_type_cumulative_button {
+        font-size: 14px;
+        margin-left: 50px;
+        background: #D8D8D8;
+        vertical-align: top;
+    }
+    #messages_by_type_ten_days_button {
+        font-size: 14px;
+        background: #F0F0F0;
+        vertical-align: top;
+    }
+    #messages_by_type_thirty_days_button {
+        font-size: 14px;
+        background: #F0F0F0;
+        vertical-align: top;
+
+    }
+    #pie_container {
+        text-align: center;
+    }
+    #pie_1 {
+        display: inline-block;
+        vertical-align: top;
+        margin: 100px 40px 0px 0px;
+    }
+    #pie_2 {
+        display: inline-block;
+        vertical-align: top;
+        margin: 100px 0px 0px 0px;
+    }
+    #pie1_total {
+        font-size: 15px;
+        text-align: left;
+        margin: 0px 0px 10px 5px;
+    }
+    #pie2_total {
+        font-size: 15px;
+        text-align: left;
+        margin: 0px 0px 10px 50px;
+    }
+    @media(max-width: 1127px) {
+        #pie_1 {
+            margin-right: 0px;
+            margin-bottom: 40px;
+        }
+    }
+
+</style>
 <body>
+<div class="sidebar">
+    <nav class="nav">
+        <p class="nav-subtitle">Message Data</p>
+        <a href="#messages_timescale_anchor" class="nav-link">Messages Sent Over Time</a>
+        <a href="#messages_by_client_anchor" class="nav-link">Messages Sent by Client</a>
+        <a href="#messages_by_type_anchor" class="nav-link">Messages Sent by Type</a>
+        <p class="nav-subtitle">User Data</p>
+        <a href="#users_anchor" class="nav-link">Number of Users</a>
```

Comment:
Our typical title style in the project right now is e.g. `Messages sent over time` -- i.e. just the first letter capitalized.

We'll also eventually want to tag all of these strings for translation; see http://zulip.readthedocs.io/en/latest/translating.html for how to do that in Jinja2 templates, but that can probably be a follow-up project.